### ISSUE TITLE
Provide a command which prints only ordinals' suffixes

### ISSUE THREAD

**Issue by dbitouze** [nlct/fmtcount#22]
Currently, `\ordinalnum{⟨number⟩}` prints `⟨number⟩` followed its suffix which depends on the underlying language.

It could be helpful for `fmtcount` to provide a public (or private) macro, say `\ordinalnumsuffix{⟨number⟩}`, that prints only the suffix of `\ordinalnum{⟨number⟩}`. For instance in English:

 - `\ordinalnumsuffix{1}` would print "<sup>st</sup>",
 - `\ordinalnumsuffix{2}` would print "<sup>nd</sup>",
 - `\ordinalnumsuffix{3}` would print "<sup>rd</sup>",
 - `\ordinalnumsuffix{4}` would print "<sup>th</sup>",
 - ...
 - `\ordinalnumsuffix{20}` would print "<sup>th</sup>",
 - `\ordinalnumsuffix{21}` would print "<sup>st</sup>",
 - `\ordinalnumsuffix{22}` would print "<sup>nd</sup>",
 - `\ordinalnumsuffix{23}` would print "<sup>rd</sup>",
 - `\ordinalnumsuffix{24}` would print "<sup>th</sup>",
 - etc.

Hence one can use for other tasks all the `fmtcount`'s machinery concerning the grammar structure and different abbreviations of the different languages.

**Comment by vincentb1** [comment]
Yes, I agree the point, that would allow to make it easier to have roman ordinals like « François Ier », so the other issue #21 that you opened on this would be easy to fulfill.

I need some more thinking about that&hellip; Anyway I would like first to be finished with issue #18 first.
